### ISSUE TITLE
Error while importing Module ovh

### ISSUE THREAD

**Issue by lauralani** [ovh/python-ovh#105]
Hi!

I'm getting the following error when trying to `import ovh` in my current project.
I'm using Python 3.10.0 on Windows 11 with ovh Verison 0.5.0.
`pip3 install ovh` says that all dependencies are satisfied, and all installed modules are up to date.

The error doesn't occur on Debian 11 with Python 3.9.2.

```
Exception has occurred: AttributeError
module 'collections' has no attribute 'MutableMapping'
  File "D:\Git\dyndns-script\dyndns_update.py", line 8, in <module>
    import ovh
```

```
Traceback (most recent call last):
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "c:\Users\lauralani\.vscode\extensions\ms-python.python-2021.11.1422169775\pythonFiles\lib\python\debugpy\__main__.py", line 45, in <module>
    cli.main()
  File "c:\Users\lauralani\.vscode\extensions\ms-python.python-2021.11.1422169775\pythonFiles\lib\python\debugpy/..\debugpy\server\cli.py", line 444, in main
    run()
  File "c:\Users\lauralani\.vscode\extensions\ms-python.python-2021.11.1422169775\pythonFiles\lib\python\debugpy/..\debugpy\server\cli.py", line 285, in run_file
    runpy.run_path(target_as_str, run_name=compat.force_str("__main__"))
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 269, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 96, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "d:\Git\dyndns-script\dyndns_update.py", line 8, in <module>
    import ovh
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\site-packages\ovh\__init__.py", line 29, in <module>
    from .client import Client
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\site-packages\ovh\client.py", line 49, in <module>
    from .vendor.requests import request, Session
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\site-packages\ovh\vendor\requests\__init__.py", line 58, in <module>
    from . import utils
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\site-packages\ovh\vendor\requests\utils.py", line 30, in <module>
    from .cookies import RequestsCookieJar, cookiejar_from_dict
  File "C:\Users\lauralani\AppData\Local\Programs\Python\Python310\lib\site-packages\ovh\vendor\requests\cookies.py", line 164, in <module>
    class RequestsCookieJar(cookielib.CookieJar, collections.MutableMapping):
AttributeError: module 'collections' has no attribute 'MutableMapping'
```

**Comment by rbeuque74** [comment]
Hello,
Sorry for the delay,
This issue will be fixed in `v1.0.0`.

**Comment by rbeuque74** [comment]
Fixed with #108.
Released in https://github.com/ovh/python-ovh/releases/tag/v1.0.0
